cve_bin_tool/checkers/__init__.py

```diff
@@ -13,5 +13,7 @@
     "libjpeg"
     "xerces",
     "libgcrypt",
-    "systemd"
+    "systemd",
+    "libsvn",
```

Comment:
Did you mean to add this libsvn line? If so there would need to be another file called libsvn